canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -1595,4 +1599,37 @@ export default class PropertiesController {
 			: 128;
 		return maxLengthForSingleLineControls;
 	}
+
+	/**
+	* Set the initial values of addRemoveRows for all structure controls
+	*/
+	setInitialAddRemoveRows() {
```

Comment:
Did this work for nested structures?

Reply:
I just added support for nested structures, thanks for reminder